addons/main/config.cpp

```diff
@@ -7,7 +7,7 @@ class CfgPatches {
         weapons[] = {};
         requiredVersion = REQUIRED_VERSION;
         requiredAddons[] = {
-            "A3_Data_F_Tank_Loadorder",
+            "A3_Data_F_Enoch_Loadorder",
             "A3_Data_F_Mod_Loadorder",
```

Comment:
Do we need this @PabstMirror ?

Reply:
I think we still do, nothing else in vanilla arma seems to require it